ophyd/status.py

```diff
@@ -257,7 +257,12 @@ def _run_callbacks(self):
         # The callbacks have access to self, from which they can distinguish
         # success or failure.
         for cb in self._callbacks:
-            cb(self)
+            try:
+                cb(self)
+            except Exception:
+                self.log.exception(
```

Comment:
Looks good, but I noticed above some things use `logger` and not `self.log` - is that intentional?

Reply:
Good catch. Looks like a failure to update things after we started using `LoggerAdapter` here. Fixed in c0fc1e7, such that the only remaining usage of `logger` is to wrap it in a `LoggerAdapter`.